cmd/upgrade_test.go

```diff
@@ -150,27 +151,28 @@ func TestCreateUpgradeCommand(t *testing.T) {
 }
 
 func TestUpgradeShouldFailForSameVersion(t *testing.T) {
+	versionToUse := common.RationalizeReleaseAndVersion(api.Kubernetes, "", "", false, false, false)
 	setupValidVersions(map[string]bool{
-		"1.10.13": true,
```

Comment:
strictly speaking this change wasn't necessary (it didn't traverse the versions allow list flow), but (IMO) it's good practice not to pin to a static version that eventually doesn't make sense (in this case 1.10.13 hasn't made sense for a long time)